src/components/Accordion/components/Default.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Colors } from '../../../essentials';
+import { Text } from '../../Text/Text';
+import { Box } from '../../Box/Box';
+import { Headline } from '../../Headline/Headline';
+import { Header } from './Header';
+import { ChevronUp } from './ChevronUp';
+import { ChevronDown } from './ChevronDown';
+import { Description } from './Description';
+import { PanelButton } from './PanelButton';
+
+interface Props {
+    label: string;
+    description?: string;
+    info?: string;
+    buttonLabel?: string;
+    expanded?: boolean;
+    children?: ReactNode;
+}
+
+const ButtonLabel = styled(Text).attrs({ as: 'p' })`
+    color: ${Colors.ACTION_BLUE_900};
+`;
+
+const PanelHeader = styled(Header)`
+    &:hover {
+        background-color: ${Colors.ACTION_BLUE_50};
+    }
+
+    &:hover ${ButtonLabel} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+
+    &:hover ${ChevronDown} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+`;
+
+const CardHeader = styled(Header).attrs({ p: '3' })`
+    background-color: ${Colors.AUTHENTIC_BLUE_50};
+    border-radius: 5px 5px 0 0;
+
+    &:hover {
+        background-color: ${Colors.ACTION_BLUE_50};
+    }
+
+    &:hover ${ButtonLabel} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+
+    &:hover ${ChevronUp} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+`;
+
+const PanelBody = styled(Box).attrs({ my: '3' })`
+    border: solid 1px ${Colors.AUTHENTIC_BLUE_200};
+    border-radius: 5px;
+`;
+
+const DefaultPanel = ({ label, description, info, buttonLabel, expanded = false, children }: Props) => {
+    const [isOpen, setIsOpen] = useState<boolean>(expanded);
+
+    return (
+        <>
+            {isOpen ? (
```

Comment:
NIT: I personally prefer to not use ternary operators with react as it can be unclear why you are rendering something.

E.G. `{isOpen && <Component />}` and `{!isOpen && <Component />}` is clearer for me but I don't know if there are any conventions around this in the codebase @nlopin

Reply:
To be honest for me is more readable with a ternary. But both approaches are equivalent

Reply:
I would personally also go for the ternary, we could mark this as resolved